app/src/main/res/values/dimens.xml

```diff
@@ -14,4 +14,7 @@
     <dimen name="dp_350">350dp</dimen>
     <dimen name="dp_30">30dp</dimen>
     <dimen name="dp_70">70dp</dimen>
-</resources>
+    <dimen name="dp_300">300dp</dimen>
+    <dimen name="dp_25">25dp</dimen>
+    <dimen name="dp_400">400dp</dimen>
+</resources>
```

Comment:
please add end of line in setting android studio
 Editor -> General -> Ensure line feed at file end on Save